mne_bids/config.py

```diff
@@ -172,21 +172,26 @@
 
 # mapping from BIDs coordinate frames -> MNE
 BIDS_TO_MNE_FRAMES = {
-    'ctf': 'ctf_head',
-    '4dbti': 'ctf_head',
-    'kityokogawa': 'ctf_head',
-    'elektaneuromag': 'head',
-    'chietiitab': 'head',
-    'captrak': 'head',
-    'acpc': 'ras',
-    'mni': 'mni_tal',
-    'fs': 'fs_tal',
-    'ras': 'ras',
-    'voxel': 'mri_voxels',
-    'mri': 'mri',
-    'unknown': 'unknown'
+    'CTF': 'ctf_head',
+    '4DBti': 'ctf_head',
+    'KitYokogawa': 'ctf_head',
+    'ElektaNeuromag': 'head',
+    'ChietiItab': 'head',
+    'CapTrak': 'head',
+    'ACPC': 'ras',  # XXX: there is no ACPC in mne-python
+    'fsaverage': 'mni_tal',
+    'MNI305': 'mni_tal',
+    'Other': 'unknown'
+}
+MNE_TO_BIDS_FRAMES = {
+    'ctf_head': 'CTF',
+    'head': 'CapTrak',
+    'mni_tal': 'fsaverage',
+    'fs_tal': 'fsaverage',  # XXX: not used
```

Comment:
What does "not used" mean here?

Reply:
@larsoner mentioned that there's no reason to use fs_Tal but it's still in mne anyways so perhaps have it map to fsaverage since that's prolly what the user meant?

Let me know if just take it out tho.

Reply:
@larsoner said:

> I wouldn’t use fs_tal, I don’t think you need it.

My suggestion: comment out the `fa_tal` line so we keep a reference, but don't "enable" it until maybe one day we find out we need it.

WDYT?

```suggestion
    # 'fs_tal': 'fsaverage',  # XXX: not used
```